compiler/pipes/check-conversions.cpp

```diff
@@ -3,16 +3,12 @@
 // Distributed under the GPL v3 License, see LICENSE.notice.txt
 
 #include "compiler/pipes/check-conversions.h"
-
-#include "common/termformat/termformat.h"
-
 #include "compiler/inferring/public.h"
 
 const std::multimap<Operation, PrimitiveType> CheckConversionsPass::forbidden_conversions = {
   {op_conv_int,      tp_Class},
   {op_conv_int_l,    tp_Class},
   {op_conv_float,    tp_Class},
-  {op_conv_string,   tp_Class},
```

Comment:
Don't you want to comment it out, leaving a hint that it's commented out intentionally?

Reply:
Added.